utils/releasetools/build-config.json

```diff
@@ -0,0 +1,36 @@
+{
+    "linux_targets": [
+        {
+          "arch": "x86_64",
+          "target": "ubuntu18.04",
+          "type": "deb",
+          "platform": "bionic"
+        },
+        {
+          "arch": "x86_64",
+          "target": "ubuntu22.04",
+          "type": "deb",
+          "platform": "focal"
+        },
+        {
+          "arch": "arm64",
+          "target": "ubuntu18.04",
+          "type": "deb",
+          "platform": "bionic"
+        },
+        {
+          "arch": "arm64",
+          "target": "ubuntu20.04",
+          "type": "deb",
+          "platform": "focal"
+        }
```

Comment:
For x86_64 we are planning to support `ubuntu18.04` and `ubuntu22.04` and for arm64 we are planning to support `ubuntu18.04` and `ubuntu20.04`. Is this inconsistency on ubuntu's end?

Reply:
We can actually support ubuntu18.04-bionic, ubuntu20.04-focal and ubuntu22.04-Jammy
It was a typo from my side

Reply:
Noble Numbat (24.04) drops this week 🤞 - is there a plan to support that too?

Reply:
@flyinprogrammer Would you mind making an issue about that. We need an overall strategy for when we start supporting new versions, and that seems like a good place to discuss it.

Reply:
👋 sorry I'm so slow to respond. Theoretically, this is at least the start of a PR: https://github.com/valkey-io/valkey/pull/971

I imagine a website edit is likely required for the [Download](https://valkey.io/download/) page. 🤷